### ISSUE TITLE
This is breaking ProjectE's EMC values for some reason.

### ISSUE THREAD

**Issue by border999** [62832/AppliedE#6]
appliede-0.11.0-beta - Removing this mod and deleting ProjectE's config restores emc values when it is readded I can't find a single item with an EMC value.
https://gist.github.com/border999/9fbe98ac5e4ade74294f87a0f30a2441 -Debug
https://gist.github.com/border999/43cdc6fe86ccf767ca54ef671763614c -Latest.log

**Comment by 62832** [comment]
This seems to be related to CC: Tweaked more than AppliedE.

```
[12May2024 14:15:15.183] [Render thread/ERROR] [moze_intel.projecte.PECore/]: Error calculating EMC values
java.lang.NoSuchMethodError: 'dan200.computercraft.shared.platform.ComponentAccess dan200.computercraft.shared.platform.PlatformHelper.createWiredElementAccess(java.util.function.Consumer)'
	at com.tom.morewires.compat.cc.CCConnectorBlockEntity.<init>(CCConnectorBlockEntity.java:74) ~[more-immersive-wires-1.20.1-1.1.3.jar%231168!/:1.1.3]
	at com.tom.morewires.compat.cc.CCWireDefinition.createBE(CCWireDefinition.java:32) ~[more-immersive-wires-1.20.1-1.1.3.jar%231168!/:1.1.3]
	at com.tom.morewires.compat.cc.CCWireDefinition.createBE(CCWireDefinition.java:21) ~[more-immersive-wires-1.20.1-1.1.3.jar%231168!/:1.1.3]
	at net.minecraft.world.level.block.entity.BlockEntityType.m_155264_(BlockEntityType.java:92) ~[client-1.20.1-20230612.114412-srg.jar%231396!/:?]
	at blusunrize.immersiveengineering.common.blocks.IEEntityBlock.lambda$new$0(IEEntityBlock.java:81) ~[ImmersiveEngineering-1.20.1-10.1.0-171.jar%231081!/:?]
	at blusunrize.immersiveengineering.common.blocks.IEEntityBlock.getClassData(IEEntityBlock.java:469) ~[ImmersiveEngineering-1.20.1-10.1.0-171.jar%231081!/:?]
	at blusunrize.immersiveengineering.common.blocks.IEEntityBlock.m_5939_(IEEntityBlock.java:366) ~[ImmersiveEngineering-1.20.1-10.1.0-171.jar%231081!/:?]
	at net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase.m_60742_(MixinBlockStateBase.java:684) ~[client-1.20.1-20230612.114412-srg.jar%231396!/:?]
	at net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase.m_60812_(MixinBlockStateBase.java:680) ~[client-1.20.1-20230612.114412-srg.jar%231396!/:?]
	at net.minecraft.world.level.block.state.BlockBehaviour.m_180643_(BlockBehaviour.java:336) ~[client-1.20.1-20230612.114412-srg.jar%231396!/:?]
	at net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase.m_60838_(MixinBlockStateBase.java:884) ~[client-1.20.1-20230612.114412-srg.jar%231396!/:?]
	at net.minecraft.world.level.block.state.BlockBehaviour$Properties.m_284108_(BlockBehaviour.java:1005) ~[client-1.20.1-20230612.114412-srg.jar%231396!/:?]
	at net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase.m_60796_(MixinBlockStateBase.java:617) ~[client-1.20.1-20230612.114412-srg.jar%231396!/:?]
	at appeng.items.parts.FacadeItem.createFacadeForItem(FacadeItem.java:167) ~[appliedenergistics2-forge-15.2.1.jar%23817!/:15.2.1]
	at gripe._90.appliede.emc.FacadeMapper.addMappings(FacadeMapper.java:46) ~[appliede-0.11.0-beta.jar%23816!/:?]
```

Would you be able to update that mod if possible and let me know if the issue persists?

**Comment by 62832** [comment]
P.S. In fact, how are you even using More Wires when there are no builds for it currently available for Forge 1.20.1 as far as I can see?

**Comment by border999** [comment]
both are up to date in my pack. Will remove Immersive Wires for the time being to test.
